pkg/kubelet/lifecycle/handlers.go

```diff
@@ -105,6 +106,30 @@ func resolvePort(portReference intstr.IntOrString, container *v1.Container) (int
 	return -1, fmt.Errorf("couldn't find port: %v in %v", portReference, container)
 }
 
+// formatURL formats a URL from args.
+func formatURL(scheme string, host string, port int, path string) *url.URL {
+	u, err := url.Parse(path)
+	// Something is busted with the path, but it's too late to reject it. Pass it along as is.
```

Comment:
i was trying to parse back if this was possible with validation code.

validateHTTPGetAction appears to check if path is not len(0), but not much else ;-/

probably another follow-up to see if we can better tighten validation on api server.